vms/rpcchainvm/vm_client.go

```diff
@@ -353,7 +353,10 @@ func (vm *VMClient) SetState(ctx context.Context, state snow.State) error {
 
 func (vm *VMClient) Shutdown(ctx context.Context) error {
 	errs := wrappers.Errs{}
-	_, err := vm.client.Shutdown(ctx, &emptypb.Empty{})
+	// Shutdown is a special case, where we want to fail fast instead of block
+	// here.
+	failFast := grpc.WaitForReady(false)
+	_, err := vm.client.Shutdown(ctx, &emptypb.Empty{}, failFast)
```

Comment:
Do we want to fail fast during shutdown? It seems like we should follow the same mechanism as the other calls. Or is this trying to handle the case that the chain creation goroutine can hang?

Reply:
I am trying to handle the case where the vm is already dead. In which case why block here vs return the error. I am going to add a little logic to turn the grpc status error into something more human readable in the case where the service is unavailable. 

The happy path will still work fine, meaning a long RPC response will not timeout. WDYT?

Reply:
I feel like this isn't really a great patch - because if the VM dies unexpectedly, we'll likely never even get to this point (as the engine could be blocked on any of the other calls). I thought the goal for this was just to ensure that we would report unhealthy if the VM had died.

Reply:
OK we can think about then. I think there needs to be a happy path where we don't fail if the server hasn't started/bootstrapped yet but otherwise fail fast. I will admit I have not dug into all the baked assumptions that exist because of the existing pattern. Will remove.